api/protobuf/api.proto

```diff
@@ -41,7 +39,7 @@ message GetFirstRequest {
 }
 
 message GetFirstResponse {
-  uint64 height = 1 [(tagger.tags) = "validate:\"required\"" ];
```

Comment:
If you want to keep using tagger, you can depend on:
```
  - buf.build/srikrsna/protoc-gen-gotag
```

If you also want to add validation codegen, then you should also add:
```
  -  buf.build/envoyproxy/protoc-gen-validate
```

And add a corresponding generation step.